static/app/views/issueDetails/streamline/eventNavigation.tsx

```diff
@@ -0,0 +1,371 @@
+  EventNavOptions.LATEST,
+];
+
+const eventDataSections: SectionDefinition[] = [
+  {
+    section: FoldSectionKey.HIGHLIGHTS,
+    label: t('Event Highlights'),
+    condition: () => true,
+  },
+  {
+    section: FoldSectionKey.STACKTRACE,
+    label: t('Stack Trace'),
+    condition: (event: Event) => event.entries.some(entry => entry.type === 'stacktrace'),
+  },
+  {
+    section: FoldSectionKey.EXCEPTION,
+    label: t('Stack Trace'),
+    condition: (event: Event) => event.entries.some(entry => entry.type === 'exception'),
+  },
+  {
+    section: FoldSectionKey.BREADCRUMBS,
+    label: t('Breadcrumbs'),
+    condition: (event: Event) =>
+      event.entries.some(entry => entry.type === 'breadcrumbs'),
+  },
+  {
+    section: FoldSectionKey.TAGS,
+    label: t('Tags'),
+    condition: (event: Event) => event.tags.length > 0,
+  },
+  {
+    section: FoldSectionKey.CONTEXTS,
+    label: t('Context'),
+    condition: (event: Event) => !!event.context,
+  },
+  {
+    section: FoldSectionKey.USER_FEEDBACK,
+    label: t('User Feedback'),
+    condition: (event: Event) => !!event.userReport,
+  },
+  {
+    section: FoldSectionKey.REPLAY,
+    label: t('Replay'),
+    condition: (event: Event) => !!getReplayIdFromEvent(event),
+  },
+];
+
+export const EventNavigation = forwardRef<HTMLDivElement, EventNavigationProps>(
+  function EventNavigation({event, group, ...props}, ref) {
+    const location = useLocation();
+    const organization = useOrganization();
+    const theme = useTheme();
+
+    const hasPreviousEvent = defined(event.previousEventID);
+    const hasNextEvent = defined(event.nextEventID);
+
+    const baseEventsPath = `/organizations/${organization.slug}/issues/${group.id}/events/`;
+
+    const jumpToSections = eventDataSections.filter(eventSection =>
+      eventSection.condition(event)
+    );
+
+    const grayText = css`
+      color: ${theme.subText};
+      font-weight: ${theme.fontWeightNormal};
+    `;
+
+    const downloadJson = () => {
+      const host = organization.links.regionUrl;
+      const jsonUrl = `${host}/api/0/projects/${organization.slug}/${group.project.slug}/events/${event.id}/json/`;
+      window.open(jsonUrl);
+      trackAnalytics('issue_details.event_json_clicked', {
+        organization,
+        group_id: parseInt(`${event.groupID}`, 10),
+      });
+    };
+
+    const {onClick: copyLink} = useCopyToClipboard({
+      successMessage: t('Event URL copied to clipboard'),
+      text: window.location.origin + normalizeUrl(`${baseEventsPath}${event.id}/`),
+      onCopy: () =>
+        trackAnalytics('issue_details.copy_event_link_clicked', {
+          organization,
+          ...getAnalyticsDataForGroup(group),
+          ...getAnalyticsDataForEvent(event),
+        }),
+    });
+
+    const {onClick: copyEventId} = useCopyToClipboard({
+      successMessage: t('Event ID copied to clipboard'),
+      text: event.id,
+    });
+
+    return (
+      <div {...props} ref={ref}>
```

Comment:
i think props here says it will be empty 
![image](https://github.com/user-attachments/assets/3c90bbd5-7ab9-42f4-8054-364c4e3266e0)

Reply:
I'll be explicit with `className`, it's meant to let the parent style it with `styled`